handlers/stations.go

```diff
@@ -588,10 +588,23 @@ func (sh StationsHandler) GetMessageDetails(c *gin.Context) {
 		return
 	}
 
-	natsMsg, err := broker.GetMessage(stationName, uint64(body.MessageSeq))
+	sm, err := sh.S.GetMessage(stationName, uint64(body.MessageSeq))
```

Comment:
what sm means?

Reply:
StoredMsg, following the terms they use inside NATS.
should I change anyway?